teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, id, age, name = 'Unknown', parent_permission: false)
+    super(id, age, name, parent_permission: parent_permission)
```

Comment:
- **[OPTIONAL]** I suggest we remove the id parameter here, in reference to my suggestion in the person class above